javascript-loops-practice/countdown.js

```diff
@@ -0,0 +1,9 @@
+/* exported countdown */
+function countdown(number) {
+  const array1 = [];
+  for (let i = 0; number >= 0; i++) {
+    array1.push(number);
+    number--;
+  }
+  return array1;
+}
```

Comment:
You're not using `i` anywhere here, so why include it? If you're going to use a `for` loop, then you should be utilizing `i` rather than number. If you don't want to use `i`, then a `while` loop would be a better choice here.